### PR TITLE
Backend/update json body

### LINKED ISSUE #38: Update expected JSON bodies in backend
Old:

```JSON
{
    "type": "...",
    ...
}
```

New:

```JSON
{
    "type": "...",
    "task": {
        ...
    }
}
```

### PR DESCRIPTION
**Pull request by loshprung**
- Created more curl scripts for testing
- Update expected JSON body for backend to match #38 
- Update `backend/README.md` correspondingly

Closes #38

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -46,11 +48,13 @@ $ php -S localhost:<port number>
...
-	"category": STRING,
-	"deadline": DATE,
-	"priority": INT
+	"params": {
+		"account_id": INT
+		"tasks_name": STRING,
+		"category": STRING,
+		"deadline": DATE,
+		"priority": INT
+	}
```
Comment (aakhare) [inline backend/README.md:85]: We're using strings for priority on the frontend so users can choose "high", "medium" , "low." 
There needs to be a completed field that would be a boolean value, so we can incorporate that logic for our progress tracker.

Comment (loshprung) [reply backend/README.md:85]: This kind of information needs to be documented somewhere. We have a design doc in our shared Google Docs. You can also append something to `frontend/README.md`. Create something formal somewhere that I can use as a guide and I will modify the task table and related functions in the backend accordingly

Comment (aakhare) [reply backend/README.md:85]: I think this information was already written in our [infrastructure plan](https://docs.google.com/document/d/1_OMUCfWQOzT0FJmh7Zsi88Ox_SkZSRT3TmfFx-i4CWQ/edit?usp=sharing), but we can talk more in person about this.

**Comment by loshprung** [comment]
New changes should resolve #44 . Note that changes introduced here may break existing functionality. Please see #44 for an overview of what has changed

**Review by maddiewaldie — APPROVED** [review]

**Review by aakhare — APPROVED** [review]

**Pull request merged**